docs/src/main/sphinx/release/release-475.md

```diff
@@ -0,0 +1,61 @@
+# Release 475 (16 Apr 2025)
+
+## General
+
+* Publish additional metrics for input tables in event listener. ({issue}`25475`)
+* Expose dynamic filter statistics in the `QueryCompletedEvent`. ({issue}`25575`)
+* Improve scalability of inline data encoding in the parallel client protocol. ({issue}`25439`)
+* Improve performance of queries involving the `exclude_columns` table function. ({issue}`25117`)
+* Disallow dropping they `system` catalog. ({issue}`24745`)
+* Fix occasional query failures when adaptive planning is enabled. ({issue}`25411`)
+* Fix incorrect results when using window functions with `DISTINCT`. ({issue}`25434`)
+* Fix query failures with `EXCEEDED_LOCAL_MEMORY_LIMIT` errors due to incorrect memory accounting. ({issue}`25600`)
+* Properly handle inline session properties for `EXPLAIN` queries. ({issue}`25496`)
+
+## Security
+
+* Fix impersonation access control when access is granted via the role. ({issue}`25166`)
+
+## BigQuery connector
+
+* Add support limiting the max parallelism via the `bigquery.max-parallelism` config property. ({issue}`25422`)
+* Fix queries getting stuck when reading large tables. ({issue}`25423`)
+
+## Delta Lake connector
+
+* Improve performance when filtering on `$path`, `$file_modified_time` and `$file_size` columns. ({issue}`25369`)
+* Improve performance of scans on delta lake tables with v2 checkpoints. ({issue}`25469`)
+
+## Hive connector
+
+* Add support for showing column comments on Hive views. ({issue}`23845`)
+* Support multiple predicate values for injected partition columns on tables using partition projection. ({issue}`17641`)
+* Fix potential failures or incorrect results when querying partitioned tables using the OpenX JSON SerDe. ({issue}`25444`)
+
+## Iceberg connector
+
+* Add `system.iceberg_tables` system table to allow listing only Iceberg tables. ({issue}`25136`)
+* Add support for [deletion vectors](https://iceberg.apache.org/spec/#deletion-vectors). ({issue}`24457`)
+* Add support for IAM role authentication REST catalog. ({issue}`25002`)
+* Add support for lock-free commit in Hive catalog. ({issue}`25445`)
+* Add support for returning column statistics for new columns in `$partitions` system table. ({issue}`25532`)
+* Optimize manifest files by the top-level partition using the `optimize_manifests` procedure. ({issue}`25378`)
```

Comment:
@ebyhr , I don't understand what this one is about.

Reply:
Previously, the procedure optimized manifest files without considering the partition spec. 

For instance, these INSERT statements create 4 manifest files:
```sql
CREATE TABLE test(a int, b int, c int) WITH (partitioning = ARRAY['b', 'c']);
INSERT INTO test VALUES (1,1,1);
INSERT INTO test VALUES (1,1,2);
INSERT INTO test VALUES (1,2,1);
INSERT INTO test VALUES (1,2,2);
```

```sql
ALTER TABLE test EXECUTE optimize_manifests;
```
The old logic rewrote to 1 manifest file.
The new logic rewrites to 2 manifest files because the top-level partition `b` has 2 values (1 or 2)

Reply:
What's the user-visible impact of this? Is it a bug fix? What happened with the table under the previous implementation? Was that a form of corruption (e.g, the table partitioning was not being respected)?

Reply:
It's not a bug fix. We expect that this way of rewriting manifest files is more optimal as consolidating to single manifest file is potentially worse for parallelism (iceberg makes parallel read per manifest file), may have higher memory usage (iceberg loads entire manifest file result in memory) and may miss out opportunities to prune manifest file reads when query filters by partition column predicate.